docs/5. Integrations/Speech to text capabilities in Glific.md

```
@@ -0,0 +1,95 @@
+To reference the eleven labs provider, pass the following additional parameters in the webhook body
+```
+"provider":"elevenlabs",
+"model":"scribe_v2"
+
+```
+<img width="678" height="484" alt="Screenshot 2026-05-08 at 4 26 45 PM" src="https://github.com/user-attachments/assets/9f35331c-7f39-44ad-9471-8042281acfa8" />
+
+Eleven labs integration does not support translation to any specified output language.
+
+## Main differences between Gemini and Eleven Labs:
+#### Translation Support
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)